src/pages/privates/details/index.jsx

```diff
@@ -0,0 +1,188 @@
+import { useState } from 'react';
+import { ChevronDownIcon } from 'lucide-react';
+import { SectionTitle } from '../../../components/section-title';
+
+export default function DetailsUser() {
+  const [isVolunteer, setIsVolunteer] = useState(false);
+
+  return (
+    <section className="pt-12 pb-20">
+      <div className="max-w-[1160px] mx-auto">
+        <div className="flex items-center justify-start">
+          <SectionTitle
+            className="pt-12 pb-14 whitespace-nowrap "
+            content="Dados Detalhados"
+          />
+          <button
+            type="submit"
+            className="ml-[740px] rounded font-bold text-4 uppercase text-social-white py-[10px]
```

Comment:
Não sei como está, mas acho que isso está errado.
Esse `ml-[740px]` não me parece a melhor forma de fazer o botão ficar corretamente espaçado.

Como o container é um flex-container, então imagino que daria para usar o `justify-between` e depois usar alguma outra propriedade para colocar as coisas no início se ainda for necessário.

Mas precisa testar. Com certeza esse `ml-[740px]` bão deveria estar aí